src/partials/query.editor.html

```diff
@@ -57,7 +57,7 @@
       <label class="gf-form-label query-keyword width-10">
         Expression
       </label>
-      <input type="text" class="gf-form-input width-20" ng-model="ctrl.target.expression" spellcheck='false' placeholder="query" ng-blur="ctrl.targetBlur()"  />
+      <input type="text" class="gf-form-input" style="width-1200px" ng-model="ctrl.target.expression" spellcheck='false' placeholder="query" ng-blur="ctrl.targetBlur()"  />
```

Comment:
this would not look good on screen sizes

Reply:
shall i just use grafana max width of 30?